src/main/2.0/scala/com/holdenkarau/spark/testing/DataFrameSuiteBase.scala

```diff
@@ -102,12 +105,11 @@ trait DataFrameSuiteBaseLike extends SparkContextProvider
   }
 
   /**
-   * Compares if two [[DataFrame]]s are equal, checks the schema and then if that
-   * matches checks if the rows are equal.
-   */
+    * Compares if two [[DataFrame]]s are equal, checks the schema and then if that
```

Comment:
So this is a general comment for the multi-line comment blocks that were changed: generally in the code base in trying to keep with Spark's style we go with
`
/**
 *
`

rather than
`
/**
  *
`